src/components/sidebar/SafeListItem/styles.module.css

```diff
@@ -2,19 +2,34 @@
   margin-top: var(--space-1);
   margin-bottom: var(--space-1);
   padding-left: var(--space-2);
-  padding-right: var(--space-2);
 }
 
 .safe {
   border-radius: 8px;
+  border: 1px solid var(--color-border-light);
 }
 
 .open {
   padding-left: 16px;
+  border: 1px solid var(--color-secondary-light);
   border-left: 6px solid var(--color-secondary-light);
   background-color: var(--color-secondary-background) !important;
 }
 
 .safe:hover {
   background-color: var(--color-secondary-background);
 }
+
+.container :global .MuiListItemSecondaryAction-root {
+  right: 16px;
+}
+
+.pendingButtons :global .MuiListItemSecondaryAction-root {
+  right: 70px;
+}
+
+@media (max-width: 400px) {
+  .pendingButtons :global .MuiListItemSecondaryAction-root {
+    right: 40px;
+  }
+}
```

Comment:
@usame-algan do you foresee incompatibilities with the bug fix you are working on?

Reply:
My idea for the bug fix was to just increase the padding since the value is already getting rounded. I think there should be no problems with the changes in this PR.